linux_docker_resources/rti_web_binaries_install_script.py

```diff
@@ -19,6 +19,7 @@
 
 
 def install_connext(installer_path, install_directory):
+
```

Comment:
Nit: unnecessary whitespace change.

Reply:
Whoops. Nice catch. That was left from a pdb session.